test/python/lcm_udpm_queue_issue_test.py

```diff
@@ -0,0 +1,81 @@
+import hashlib
+import os
+import unittest
+import time
+
+from lcm import LCM
+
+
+def lcm_handle_all(lcm, timeout=0.01):
+    total_count = 0
+    while True:
+        count = lcm.handle_timeout(timeout)
+        if count == 0:
+            return total_count
+        total_count += count
+    return total_count
+
+
+def unique_lcm_url(s):
+    """Returns a unique LCM url given a string."""
+    rand = [ord(c) for c in hashlib.sha256(s).digest()]
+    return "udpm://239.{:d}.{:d}.{:d}:{:d}?ttl=0".format(
+        rand[0], rand[1], rand[2], 20000 + rand[3])
```

Comment:
This doesn't work for Python3. Looks like it should be

```suggestion
def unique_lcm_url(s):
    """Returns a unique LCM url given a string."""
    rand = [c for c in hashlib.sha256(s.encode()).digest()]
    return "udpm://239.{:d}.{:d}.{:d}:{:d}?ttl=0".format(
        rand[0], rand[1], rand[2], 20000 + rand[3])
```

However, I'd prefer to factor out this randomness. A hard-coded URL would be preferable in tests to reduce inconsistent results. That being said, I don't think it's likely to be an issue in this case.